src/features/cohortes/CohortSelector.tsx

```diff
@@ -69,15 +69,16 @@ const areCohortOrgaAccessRequestPendingSelector = createSelector(
 const showCreateAccessRequestAlertSelector = createSelector(
   [self, cohortOrgaRepartitionDataSelector, orgaIdsOutOfPractitionerPerimeterSelector],
   (state, orgaRepartitionData, orgaIdsOutOfPractitionerPerimeter) => {
-    const { totalPatients, cohort } = state.exploredCohort
+    const { originalPatients, cohort } = state.exploredCohort
+    const patientsInPerimeterCount = originalPatients?.length
     const cohortMembersCount = cohort?.member?.length
 
     if (orgaRepartitionData && orgaIdsOutOfPractitionerPerimeter) {
       for (const { id: orgaId } of orgaRepartitionData) {
         if (orgaId && orgaIdsOutOfPractitionerPerimeter.includes(orgaId)) {
           /*  If group members count differs from originalPatients count,
            *  this means there are patients out of the practitioner's perimeter */
-          return totalPatients !== cohortMembersCount
+          return patientsInPerimeterCount !== cohortMembersCount
```

Comment:
what if:
- We explore a cohort with 3 perimeters: A (1 patient), B (1 patient) , C (0 patient)
- I have access only to A
- This condition will return true for both B and C but it should return true only for B (since there are no patients in C) right ?

Reply:
this can happen indeed but we don't have a workaround for this